### PR TITLE
Update Ubuntu Install Instructions

### PR DESCRIPTION
**Pull request by spomorski**
Updating Ubuntu install instructions together with David.

### CODE REVIEW

File: site/docs/install-ubuntu.md [hunk 1]
```diff
@@ -28,24 +28,24 @@ The binary installers are on Bazel's [GitHub releases page](https://github.com/b
...
-
-**Note:** In the installer file names listed in this document, replace
-`<version>` with the appropriate Bazel version number.
+### Step 2. Download Bazel
 
-Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases).
+Next, download the Bazel binary installer named `bazel-<version>-installer-linux-x86_64.sh` from the [Bazel releases page on GitHub](https://github.com/bazelbuild/bazel/releases).
 
-Download the binary installer `bazel-<version>-installer-linux-x86_64.sh`.
+**Note:** In the installer file name listed in this document, replace
```
Comment (laszlocsomor) [inline site/docs/install-ubuntu.md]: It's common practice, for example in manpages, that the user has to replace fields in &lt;angle_brackets&gt;, so I suggest either dropping this sentence or appending it to the last sentence in parentheses, e.g.:

> Next, download the Bazel binary installer named `bazel-<version>-installer-linux-x86_64.sh` from the [Bazel releases page on GitHub](https://github.com/bazelbuild/bazel/releases) (where &lt;version&gt; is a Bazel version, e.g. "0.12.0").

File: site/docs/install-ubuntu.md [hunk 2]
```diff
@@ -68,28 +68,30 @@ export PATH="$PATH:$HOME/bin"
 
 You can also add this command to your `~/.bashrc` file.
 
-_<sup>1</sup>Note that Bazel includes an embedded JDK 8, which can be used even if a JDK is already
+_<sup>1</sup>Note that Bazel includes an embedded JDK, which can be used even if a JDK is already
 installed. `bazel-<version>-without-jdk-installer-linux-x86_64.sh` is a version of the installer 
-without embedded JDK 8. Only use this installer if you already have JDK 8 installed._
+without an embedded JDK. Only use this installer if you already have JDK 8 or later installed._
```
Comment (laszlocsomor) [inline site/docs/install-ubuntu.md]: No, later JDKs are not supported, only JDK 8 is.

File: site/docs/install-ubuntu.md (hunk 1 above)
Comment (spomorski) [reply site/docs/install-ubuntu.md]: Removed the note.

File: site/docs/install-ubuntu.md (hunk 2 above)
Comment (spomorski) [reply site/docs/install-ubuntu.md]: Updated and added an explicit note to reflect this.

**Comment by laszlocsomor** [comment]
LGTM

**Pull request closed without merging**